template.tpl

```diff
@@ -41,19 +41,12 @@ ___TEMPLATE_PARAMETERS___
           100
         ],
         "type": "STRING_LENGTH"
-      },
-      {
-        "args": [
-          "t2_[a-z0-9]+"
-        ],
-        "type": "REGEX"
       }
     ],
-    "displayName": "Advertiser ID",
+    "displayName": "Ad Account ID",
     "simpleValueType": true,
     "name": "id",
     "type": "TEXT",
```

Comment:
sn00bie question: is the trailing comma here OK?  That is, that's not valid JSON, but it is valid JavaScript.  (Was this file autogenerated, or edited by hand?)

Reply:
Autogenerated!

Reply:
Excellent question!

Reply:
Good catch, I tried the replacement again and it should be fixed now. Bad copy pasta, sorry!